docs/book/index.html

```diff
@@ -2,7 +2,7 @@
   <div class="jumbotron">
```

Comment:
This file can be removed, because the homepage of the documentation is generated automatically.

The configuration file for docs (`mkdocs.yml`) needs an update instead.

https://github.com/zendframework/zend-expressive-swoole/blob/e7d2700a6d9bd7193551b7bbda983b283a62fc98/mkdocs.yml#L6-L7

Reply:
@froschdesign I removed the `docs/book/index.html` file. The `mkdocs.yml` was already updated.